### PR TITLE
feature/extra-constols-jsx

### PR DESCRIPTION
**Pull request by rheinardkorf**
Addresses JS requirements for:
#10 
#11 
#12 
#13 
#14 
#16 
#17 

Further comments will be made on each of those tickets.

### CODE REVIEW

**Comment by rheinardkorf** [comment]
Please see correction, this does not yet implement #15 (the image control).

File: php/endpoints/class-preview.php [hunk 1]
```diff
@@ -60,8 +60,11 @@ public function mock_attributes( $block ) {
 				case 'textarea':
 					$value = '<' . wp_json_encode( $field['name'] ) . '>';
 					break;
+//				case 'toggle':
+//					$value = '<' . wp_json_encode( $field['name'] ) . '>';
+//					break;
```
Comment (lukecarbis) [inline php/endpoints/class-preview.php]: Why are these lines commented out?

**Review by lukecarbis — APPROVED** [review]
Looks good. If there's a good reason for the lines to be commented out, then go ahead and merge. Otherwise, uncomment, or just delete the lines?

File: php/endpoints/class-preview.php (hunk 1 above)
Comment (rheinardkorf) [reply php/endpoints/class-preview.php]: I will actually remove this block. Its only there for when there are significant variances which I have not yet come across as I was building out the controls.

**Pull request merged**